adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -121,14 +130,11 @@ class Transform(protected val args: TransformArgs) extends BDGSparkCommand[Trans
     if (args.recalibrateBaseQualities) {
       log.info("Recalibrating base qualities")
       val knownSnps: SnpTable = createKnownSnpsTable(sc)
-      val stringency = if (args.strictBQSR) {
```

Comment:
I'm not sure how I feel about collapsing import stringency and BQSR stringency down into a single flag. The two aren't logically tied together.

Is your thought here that the two stringencies (aside: what is the plural of stringency? I think that I am doing English wrong here...) are independent? I.e., you will only set the stringency flag on import if you are reading from FASTQ, which is unaligned data. If you're running BQSR, your data must be aligned, therefore the function of the stringency flag is context dependent?

Let me know if what I said wasn't clear...